test/azul_test_case.py

```diff
@@ -135,6 +135,13 @@ class RE(str):
                 ),
 
                 'Call to deprecated function (or staticmethod) patch_source_cache',
+
+                # FIXME: Remove after upgrading to a v8 Elasticsearch client
+                #        https://github.com/elastic/elasticsearch-py/issues/1698
```

Comment:
FIXMEs have to refer to Azul issues, and those issues can't already be closed.

Do these deprecation warnings show up in a deployed Lambda as well? If so, they will probably be very prevalent, and therefore cause lots of noise. Please raise this in PL.